### PR TITLE
[BULK] - DocuTune remediation - Sensitive terms with GUIDs (part 2)

### PR DESCRIPTION
**Pull request by meganbradley**
Applying sensitive terms with GUID changes as part of Content SFI and outlined in [Overview - Writing content securely - Platform Manual](https://review.learn.microsoft.com/en-us/help/platform/security-reference?branch=main). Changes are part of the Microsoft-wide SFI effort. Point of contact: @CelesteDG

DocuTune v1.5.2.0
CorrelationId: [ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db](https://github.com/Azure/azure-docs-powershell-azuread/pulls?q=is%3Apr+ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db+)

#docutune

### CODE REVIEW

**Comment by meganbradley** [comment]
#docutune-review

File: azureadps-2.0-preview/AzureAD/Get-AzureADApplication.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Output:
 
     ObjectId                             AppId                                DisplayName
     --------                             -----                                -----------  
-    ed192e92-84d4-4baf-997d-1e190a81f28e 36ee4c6c-0812-40a2-b820-b22ebd02bce3 MyNewApp
+    00001111-aaaa-2222-bbbb-3333cccc4444 36ee4c6c-0812-40a2-b820-b22ebd02bce3 MyNewApp
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Get-AzureADApplication.md:60]: AppId value not updated.

File: azureadps-2.0-preview/AzureAD/Get-AzureADApplication.yml [hunk 2]
```diff
@@ -27,7 +27,7 @@ examples:
   summary: ""
 - title: 'Example 2: Get an application by ID'
   code: |-
-    PS C:\>Get-AzureADApplication -Filter "AppId eq 'ed192e92-84d4-4baf-997d-1e190a81f28e'"
+    PS C:\>Get-AzureADApplication -Filter "AppId eq '00001111-aaaa-2222-bbbb-3333cccc4444'"
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Get-AzureADApplication.yml:30]: GUIDs in the output haven't been updated.

File: azureadps-2.0-preview/AzureAD/Get-AzureADApplicationExtensionProperty.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ The **Get-AzureADApplicationExtensionProperty** cmdlet gets application extensio
 
 ### Example 1: Get extension properties
 ```
-PS C:\>Get-AzureADApplicationExtensionProperty -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84"
+PS C:\>Get-AzureADApplicationExtensionProperty -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Get-AzureADApplicationExtensionProperty.md:30]: Output GUIDs haven't been updated.

File: azureadps-2.0-preview/AzureAD/Get-AzureADApplicationExtensionProperty.yml [hunk 4]
```diff
@@ -16,7 +16,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Get extension properties'
   code: |-
-    PS C:\>Get-AzureADApplicationExtensionProperty -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84"
+    PS C:\>Get-AzureADApplicationExtensionProperty -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Get-AzureADApplicationExtensionProperty.yml:19]: Output GUIDs haven't been updated.

**Review by msewaweru — CHANGES_REQUESTED** [review]
A few suggestions added.

File: azureadps-2.0-preview/AzureAD/Get-AzureADApplication.md (hunk 1 above)
Comment (cilwerner) [reply azureadps-2.0-preview/AzureAD/Get-AzureADApplication.md:60]: DocuTune is catching AppId in L51 and sees the same value in L60, but associated with object Id. The notation here is odd, so DocuTune is a bit off here.

File: azureadps-2.0-preview/AzureAD/Get-AzureADApplicationOwner.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ The **Get-AzureADApplicationOwner** cmdlet get an owner of an Azure Active Direc
 
 ### Example 1: Get the owner of an application
 ```
-PS C:\>Get-AzureADApplicationOwner -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84"
+PS C:\>Get-AzureADApplicationOwner -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADApplicationOwner.md:29]: Same here...

**Review by cilwerner — COMMENTED** [review]
@msewaweru, I'm unsure why there's so many misses here. Between a combination of the ymls and output tables, DocuTune is misbehaving.

I'll continue looking through the others now that I have a bit of time, some manual edits may be needed here if DocuTune can't catch them all.

False positives are more concerning.

**Review by msewaweru — APPROVED** [review]
Good to go, @CelesteDG.

**Pull request merged**